search/searcher/search_term_prefix.go

```diff
@@ -35,5 +35,10 @@ func NewTermPrefixSearcher(indexReader index.IndexReader, prefix string,
 		tfd, err = fieldDict.Next()
 	}
```

Comment:
there is another error unchecked here, using this pattern, we must check for non nil err upon exiting the loop, as we may have broken out of the for loop due to an error.

Reply:
Done!